### PR TITLE
[W5.11][F11-A3]Woody Lau

### PR DESCRIPTION
**Pull request by WoodyLau**
Added a sort command that lists contacts sorted alphabetically. Updated tests and documentation to match.

### CODE REVIEW

**Comment by ghost** [comment]
Looks great! Complete the coding, documentation and testing 👍

File: doc/UserGuide.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sorting all persons : `sort`
```
Comment (OuyangDanwen) [inline doc/UserGuide.md:56]: Good!

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class SortCommandTest {
```
Comment (OuyangDanwen) [inline test/java/seedu/addressbook/commands/SortCommandTest.java:13]: Kudos for adding Junit tests!

File: src/seedu/addressbook/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Lists all persons in the address book to the user.
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/commands/SortCommand.java:11]: Is this the correct description of the class?

**Review by OuyangDanwen — COMMENTED** [review]
Generally great work with the addition of Junit tests! One thing to take note: commit message should be in imperative tone, i.e. start with a verb in present tense. You can close the PR after viewing the comments. @WoodyLau

**Pull request closed without merging**